controls/radrichtextbox/document-elements/table/working-with-tables.md

```diff
@@ -0,0 +1,353 @@
+---
+title: Working with Tables in Code
+page_title: Working with Tables in Code
+description: Working with Tables in Code
+slug: radrichtextbox-working-with-tables
+tags: table
+published: True
+position: 1
+---
+
+# Working with Tables in Code
+
+This article shows you can create or retrieve tables in the code behind. 
+
+## Creating a Table Programmatically
+
+The __RadRichTextBox__ exposes a rich API, which allows you to use various methods to add, modify or delete elements from the [__RadDocument__]({%slug radrichtextbox-features-document-elements-raddocument%}). The methods exposed by the API can be wired to a UI and get executed upon user interaction with this UI.
```

Comment:
The methods exposed by the API can be wired to a UI and get executed upon user interaction with this UI. - that doesn't seem related to the information discussed here.